src/applications/mhv-medical-records/components/LabsAndTests/PathologyDetails.jsx

```diff
@@ -114,12 +121,24 @@ ${record.results} \n`;
 
       <div className="test-details-container max-80">
         <h2>Details about this test</h2>
+        <h3 className="vads-u-font-size--md vads-u-font-family--sans">
+          Date and time collected
+        </h3>
+        <p data-testid="pathology-date" data-dd-privacy="mask">
+          {record.date}
+        </p>
```

Comment:
The `DateSubheading` is also printing "Date and time collected". That one uses `record.dateCollected` and this one uses `record.date`. What is the difference between these? Which is correct? Should the field be duplicated?

Reply:
Good catch! I added it after adding it to the PDF but didn't consider it was also above.